lib/pymedphys/_streamlit/apps/metersetmap/_trf.py

```diff
@@ -41,9 +41,8 @@ def _get_mosaiq_configuration(headers):
     return machine_centre_map, mosaiq_details, mosaiq_servers
 
 
-@st.cache
 def get_logfile_mosaiq_info(
-    headers, machine_centre_map, mosaiq_details, mosaiq_servers
+    headers: "pd.DataFrame", machine_centre_map, mosaiq_details, mosaiq_servers
```

Comment:
I wasn't aware you could quote a type hint.

Reply:
Yup, that means you don't need pd to be defined at import time :). Win :)

Reply:
That is huge!
Especially given the somewhat tricky approach to import in PyMedPhys.
Ok, so now I can use type hinting more easily and extensively.

Reply:
The only gotchya, is pylint gets a little confused. For example:

```python
import pathlib  # <-- pylint complains of 'unused import'... just need to override.

def a_function(a_variable: "pathlib.Path"):
    do_something()
```

Reply:
> Ok, so now I can use type hinting more easily and extensively.

As we begin to use it more, we should roll out a mypy check within the CI also.